src/spikeinterface/postprocessing/tests/test_template_similarity.py

```diff
@@ -8,7 +8,9 @@
 class TestSimilarityExtension(AnalyzerExtensionCommonTestSuite):
 
     def test_extension(self):
-        self.run_extension_tests(ComputeTemplateSimilarity, params=dict(method="cosine_similarity"))
+        self.run_extension_tests(ComputeTemplateSimilarity, params=dict(method="cosine"))
+        self.run_extension_tests(ComputeTemplateSimilarity, params=dict(method="l2"))
+        self.run_extension_tests(ComputeTemplateSimilarity, params=dict(method="l1", max_lag_ms=0.2))
```

Comment:
You did not adopt the beautifull pytest parametrize ?

Reply:
you're changing camarade